Makefile

```diff
@@ -1,4 +1,4 @@
-GO_VERSION = 1.16
+GO_VERSION = 1.17
```

Comment:
why not 1.18?

Reply:
SInce operator-sdk is still using Go 1.17 and K8s 1.23. I think it would be better to wait for operator-sdk v.1,21 which probably have these changes.

Reply:
I would prefer to use whatever OCP and k8s use in the next release, but I leave it up to you for NMO (NHC already is on 1.18)